xdsl/dialects/builtin.py

```diff
@@ -423,35 +423,66 @@ def verify(self) -> None:
 AnyIntegerAttr: TypeAlias = IntegerAttr[IntegerType | IndexType]
 
 
+class _FloatType(ParametrizedAttribute, TypeAttribute, ABC):
+    width: ParameterDef[IntAttr]
+
+    def __init__(self, width: int | IntAttr) -> None:
+        if isinstance(width, int):
+            width = IntAttr(width)
+        super().__init__([width])
+
+    @property
+    def get_bitwidth(self) -> int:
```

Comment:
My personal preference is for fewer helpers, especially in cases where typing out the implementation takes fewer characters than the name of the helper. I would lean towards removing this property, and having clients use `attr.width.data` directly. WDYT?

Reply:
Can do.